pylint/pyreverse/main.py

```diff
@@ -112,8 +117,7 @@
         dict(
             short="m",
             default=None,
-            type="yn",
-            metavar="<y or n>",
+            action="store_true",
```

Comment:
Ain't it a breaking change ?

Reply:
Imo it means we goes from having to do ``-m y`` or  ``-m=n``   to simply ``-m`` with a parsing error for the former.

Reply:
Ah that might be true. I'll investigate and add a test this afternoon.

Reply:
But I do agree that not having y/n options is better. Not sure if asking user to upgrade their scripts is worth it even in 3.0 or another breaking change. Maybe allowing both behavior would be preferable, but ti's not going to make our maintenance burden lighter.

Reply:
Added a regression test!